index.bs

```diff
@@ -2246,6 +2249,13 @@ the [RP] know that the extension was honored by the client. Similarly, any exten
 specify a method of augmenting the [=authenticator data=] to let the [RP] know that the extension was honored by the
 authenticator.
 
+If an extension is designed to enable the [RP] to know whether it was understood and acted on,
+the extension MUST return some data from the client to the [RP] for this purpose.
+If an extension does not otherwise require any result values, it SHOULD be defined as returning a Boolean extension result,
+set to `true` to signify that the extension was understood and processed.
+Likewise, such an extension that is processed by the authenticator that does not otherwise require any result values
+SHOULD also return a Boolean extension result,
+set to `true` to signify that the extension was understood and processed.
```

Comment:
This paragraph seems redundant with the preceding one.

IMO there is no such thing as an extension that is not designed to enable the RP to know if it was understood and acted on. The rest of the paragraph is just recommending returning "true" as a specific way of letting the RP know the extension was honored (the latter part was specified in the previous paragraph). If we do want to add this, it seems like it could be added to the previous paragraph.